### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by deprilula28**

### CODE REVIEW

File: src/main/kotlin/com/coalesce/bot/commands/executors/Request.kt [hunk 1]
```diff
@@ -0,0 +1,170 @@
...
+import net.dv8tion.jda.core.entities.PrivateChannel
+import net.dv8tion.jda.core.entities.Role
+import net.dv8tion.jda.core.entities.User
+import net.dv8tion.jda.core.events.message.react.MessageReactionAddEvent
+import java.io.IOException
+import java.io.OutputStreamWriter
+import java.net.URL
+import java.util.*
+
+class Request(@Inject val bot: Main) {
```
Comment (Proximyst) [inline src/main/kotlin/com/coalesce/bot/commands/executors/Request.kt]: Replace it with
`@Inject constructor(val bot: Main)`

File: src/main/kotlin/com/coalesce/bot/commands/executors/Request.kt [hunk 2]
```diff
@@ -0,0 +1,170 @@
...
+                .addField("Authorize your GitHub Account",
+                        "https://github.com/login/oauth/authorize?client_id=2e8a8ed194265736c99a&scope=user&state" +
+                                "=${role.id},${user.id}", false)
+                .build()).queue()
+
+        globalChannel.sendMessage(user.asMention + " I've sent you a private message with instructions.").queue()
+    }
+}
+
+class ValidateRequest(@Inject val bot: Main) {
```
Comment (Proximyst) [inline src/main/kotlin/com/coalesce/bot/commands/executors/Request.kt]: Replace with:
`@Inject constructor(val bot: Main)`

**Review by NahuLD — APPROVED** [review]

**Review by Proximyst — CHANGES_REQUESTED** [review]
Fix constructors

File: src/main/kotlin/com/coalesce/bot/commands/executors/Request.kt (hunk 1 above)
Comment (deprilula28) [reply src/main/kotlin/com/coalesce/bot/commands/executors/Request.kt]: are you serious

**Review by Proximyst — APPROVED** [review]
Will now actually run.

**Pull request merged**